apps/collabora_integrated.sh

```diff
@@ -46,16 +46,6 @@ then
 We will now remove the old docker and install the app from Nextcloud instead."
     # Remove docker image
     docker_prune_this 'collabora/code'
-    # Disable RichDocuments (Collabora App) if activated
-    if is_app_installed richdocuments
-    then
-        nextcloud_occ app:remove richdocuments
-    fi
-    # Disable OnlyOffice (Collabora App) if activated
-    if is_app_installed onlyoffice
-    then
-        nextcloud_occ app:remove onlyoffice
-    fi
```

Comment:
Why did you remove both of these?

Reply:
because they already get removed down below: https://github.com/nextcloud/vm/pull/1989/files#diff-9307a1a930d542adcec66849575564e9e6def7462586ff5a71da9c5eb985dc17R124-R135

Reply:
https://github.com/nextcloud/vm/blob/1caa815a8a2910cabfb9d3e19cb5fdff46bb6083/apps/collabora_integrated.sh#L124-L134

Reply:
ping @enoch85